code/modules/admin/holder2.dm

```diff
@@ -11,6 +11,11 @@ GLOBAL_PROTECT(admin_datums) // This is protected because we dont want people ma
 	/// Our currently linked marked datum
 	var/datum/marked_datum
 
+	/// TEMPORARY UNTIL WE FIND A BETTER PLACE TO PUT THIS. IF THIS IS MERGED, BLOW UP ALL CODE REVIEWERS
+	var/team_switch_tab_index = 1
+	// TODO TO DO ctodo
+	// LOOK AT ME!!!
```

Comment:
Reviewers, if you can think of a nice place to put this, please let me know

Reply:
Hi hello, I will not be blown up. Honestly, given the way the admin UI works out, I don't think this would be the worst place to put this? The alternative might be an associative list by admin holder on the team datum itself, but I don't think that'd be too much better?